doc/admin/deployment.md

```diff
@@ -1227,7 +1238,11 @@ log\_file: /tmp/daos\_agent2.log
 ## Agent Startup
 --------------
 
-DAOS Agent is a standalone application to be run on each compute node. It can be configured to use secure communications (default) or can be allowed to communicate with the control plane over unencrypted channels. The example below for executing daos_agent specifies to operate in insecure mode as certificate support is not fully integrated into DAOS as of 0.6
+DAOS Agent is a standalone application to be run on each compute node.
+It can be configured to use secure communications (default) or can be allowed
+to communicate with the control plane over unencrypted channels. The example
+below for executing daos_agent specifies to operate in insecure mode as
+certificate support is not fully integrated into DAOS as of 0.6
```

Comment:
Missing a trailing period to end the sentence. Also the phrasing is a little awkward. Maybe, "The following example shows daos_agent being configured to operate in insecure mode due to incomplete integration of certificate support as of the 0.6 release." ?